test/token/ERC721/extensions/ERC721Votes.test.js

```diff
@@ -9,8 +9,14 @@ const { shouldBehaveLikeVotes } = require('../../../governance/utils/Votes.behav
 
 contract('ERC721Votes', function (accounts) {
   const [ account1, account2, account1Delegatee, other1, other2 ] = accounts;
-  this.name = 'My Vote';
+  const name = 'My Vote';
   const symbol = 'MTKN';
+  const tokens = [
+    '10000000000000000000000000',
+    '10',
+    '20',
+    '30',
+  ].map(web3.utils.toBN);
```

Comment:
Because of the other arguments passed from `map` (index, etc.) we should do this to be sure they aren't affecting the result:

```suggestion
  ].map(n => web3.utils.toBN(n));
```